manageiq-operator/pkg/apis/manageiq/v1alpha1/manageiq_types.go

```diff
@@ -60,6 +58,7 @@ type ManageiqSpec struct {
 	PostgresqlSharedBuffers  string `json:"postgresqlSharedBuffers"`
 
 	// kafka deployment information
+	// +optional
```

Comment:
I guess this is some generator directive to show that `KafkaSecret` is optional? It looks strange that it's at the same level as the heading comment; it makes it seem like the entire group is optional. Is there some other way to do this?

Reply:
Done, moved this one as the last parameter.